reactDashboard/src/components/RouteMapRules/NationalRouteMapRules.jsx

```diff
@@ -30,4 +30,4 @@ const RouteMapRules = () => {
   );
 };
 
-export default RouteMapRules;
+export default NationalRouteMapRules;
```

Comment:
Changed name to differentiate between this component and the new regional one.

Reply:
I think these could be refactored into one re-useable component